ecommerce/android/README.md

```diff
@@ -0,0 +1,8 @@
+# CMPE451 - E-commerce app - Android
+
+# Instructions
+
+1. `git pull`
+2. `cd ecommerce/android/`
+3. Make sure you have installed Android Studio and run the project folder(Getlix) on it.
```

Comment:
I updated the README.md now according to your suggestion.

The initial design of the splash screen is temporary, it needs to be updated with a better design. The colors are also temporary since we haven't decided on a color palette as a group yet.